resources/resources_handler.go

```diff
@@ -11,14 +12,17 @@ const resourcesPath = "keda.yaml"
 const olmResourcesPath = "keda-olm-operator.yaml"
 const LastConfigID = "olm-operator.keda.sh/last-applied-configuration"
 
-func GetResourcesManifest() (mf.Manifest, error) {
+func GetResourcesManifest(kedaRelease string) (mf.Manifest, error) {
```

Comment:
can we please add a test and a validation that `kedaRelease` is correct?

Reply:
I decided to use `semver` package to do the test, but there's an issue. Keda releases on GH are versioned using semantic versioning scheme, but the build arg used during docker build is not (missing `'v'` prefix). See for example in keda logs:
```
2024-02-20T13:20:20Z	INFO	setup	KEDA Version: 2.12.0
```

Should users use `kedaRelease: "v2.12.1"` or `kedaRelease: "2.12.1"`?
Or should we add the `v` if it's not present? 

This will also determin the dir structure.

Reply:
I would go with both options, default (documented) would be `kedaRelease: "2.12.1"` and prepend `v` manually